lib/Elastica/Type/Mapping.php

```diff
@@ -294,4 +294,5 @@ public static function create($mapping)
 
         return $mappingObject;
     }
+
```

Comment:
can you revert such changes that haven't any affects? also in previous file.

to make diff more clear